modules/carto/src/sources/h3-query-source.ts

```diff
@@ -0,0 +1,48 @@
+/* eslint-disable camelcase */
+import {CartoBaseSource} from './base-source';
+import {
+  CartoAggregationOptions,
+  CartoQuerySourceOptions,
+  CartoSourceOptions,
+  TilejsonSource
+} from './common';
+
+export type CartoH3QuerySourceOptions = CartoSourceOptions &
+  CartoQuerySourceOptions &
+  CartoAggregationOptions;
+type UrlParameters = {
+  aggregationExp?: string;
+  aggregationResLevel?: string;
+  geo_column?: string;
+  q: string;
+  queryParameters?: string;
+};
+
+const CartoH3QuerySource: TilejsonSource<CartoH3QuerySourceOptions> = async function (
+  options: CartoH3QuerySourceOptions
+): Promise<any> {
+  const {
+    aggregationExp = '1 AS value',
```

Comment:
@felixpalmer, I would like to know, why this default value 🤔.  Below you are checking if the value exist, it's always true by default and I don't know If this default value can hide possible errors by not passing the parameter from outside

Reply:
In our current code we provide this as a default, perhaps it is better to remove it to make it clearer to the user they need to provide the expression. After all just aggregating using `1 AS value` isn't very useful.

I'll remove the default value and mark it as required

Reply:
I agree with @felixpalmer, the user must provide the expression